public_html/wp-content/mu-plugins/blocks/includes/components/post-list.php

```diff
@@ -43,7 +43,7 @@ function render_post_list( array $rendered_items, $layout = 'list', $columns = 1
 	?>
```

Comment:
`MUST` - I think this should this be `has-layout-{layout}` instead, since it'd be considered a state.

https://github.com/WordPress/wordcamp.org/pull/123/files#diff-edfb05ceecdc904d955b60b5e64046bcL27

Reply:
Fixed in https://github.com/WordPress/wordcamp.org/pull/123/commits/47f250ec5776657736fcb324c48d36530322c92f